gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -4,53 +4,91 @@
+              <NxStatefulTextInput validator={validator}/>
+            </label>
+          </div>
+          <div className="nx-form-group">
+            <label className="nx-label nx-label--optional">
+              <span className="nx-label__text">Label</span>
+              <NxStatefulTextInput/>
+            </label>
+          </div>
         </div>
-        <div className="nx-form-group">
-          <label className="nx-label">
-            <span className="nx-label__text">Second column</span>
-            <span className="nx-sub-label">This is a sub-label.</span>
-            <input type="text" className="nx-text-input"/>
-          </label>
-        </div>
-        <div className="nx-form-group">
-          <button className="nx-btn" type="button">Button</button>
-        </div>
-      </div>
-      <div className="nx-form-row">
         <fieldset className="nx-fieldset">
-          <legend className="nx-label nx-label--optional">Checkboxes</legend>
-          <NxCheckbox isChecked={false}>Checkbox 1</NxCheckbox>
-          <NxCheckbox isChecked={true}>Checkbox 2</NxCheckbox>
-          <NxCheckbox isChecked={false}>Checkbox 3</NxCheckbox>
+          <legend className="nx-legend">
+            <span className="nx-legend__text">
+              Checkboxes
+            </span>
+          </legend>
+          <NxCheckbox onChange={toggleRed} isChecked={isRed}>Red</NxCheckbox>
+          <NxCheckbox onChange={toggleBlue} isChecked={isBlue}>Blue</NxCheckbox>
+          <NxCheckbox onChange={toggleGreen} isChecked={isGreen}>Green</NxCheckbox>
         </fieldset>
         <fieldset className="nx-fieldset">
-          <legend className="nx-label">Radio buttons</legend>
-          <NxRadio name="demo1" value="demo1" isChecked={true}>Radio Button 1</NxRadio>
-          <NxRadio name="demo2" value="demo2" isChecked={false}>Radio Button 2</NxRadio>
-          <NxRadio name="demo3" value="demo3" isChecked={true}>Radio Button 3</NxRadio>
+          <legend className="nx-legend"><span className="nx-legend__text">Radio buttons</span></legend>
+          <NxRadio name="color"
+                   value="red"
+                   onChange={setColor}
+                   isChecked={color === 'red'}
+                   radioId="color-red">
+            Red
+          </NxRadio>
+          <NxRadio name="color"
+                   value="purple"
+                   onChange={setColor}
+                   isChecked={color === 'purple'}
+                   radioId="color-purple">
+            Purple
+          </NxRadio>
+          <NxRadio name="color" value="green" onChange={setColor} isChecked={color === 'green'} radioId="color-green">
+            Green
+          </NxRadio>
+          <NxRadio name="color" value="blue" onChange={setColor} isChecked={color === 'blue'} radioId="color-blue">
+            Blue
+          </NxRadio>
         </fieldset>
       </div>
+      <footer className="nx-tile-footer">
```

Comment:
`nx-form-footer`? Noticed becase the margins are not removed from `nx-btn-bar`. Also, should it remove margins from the alert?

Reply:
> `nx-form-footer`? Noticed becase the margins are not removed from `nx-btn-bar`. Also, should it remove margins from the alert?

I considered this but I think because we're demonstrating a form/tile hybrid and we use only tile styling elsewhere it should be the tile footer. I will be fixing the discrepancies between the various footers.

Reply:
> > `nx-form-footer`? Noticed becase the margins are not removed from `nx-btn-bar`. Also, should it remove margins from the alert?
> 
> I considered this but I think because we're demonstrating a form/tile hybrid and we use only tile styling elsewhere it should be the tile footer. I will be fixing the discrepancies between the various footers.

I removed the whole form as tile example, I decided that it was too complicated: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/af777406b35c1158e36713e2935d2b2a6b76a5da